nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -292,6 +305,74 @@ constexpr int blockage_test_ongoing = 2;
 constexpr int min_odometry_hz = 10;
 constexpr int max_odometry_hz = 50;
 
+struct FieldInfo
+{
+  FieldInfo(
```

Comment:
For some reason, clang-tidy interprets this as a possibly throwing constructor.... can you mark it with `noexcept` just to silence the warning?

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/1b1d26bcc5ee0417fb1837276062016dddc870ad